x/oracle/spec/01_concepts.md

```diff
@@ -0,0 +1,31 @@
+<!--
+order: 1
+-->
+
+# Concepts
+
+The oracle module is very minimal, but users should understand what the `Oracle` is and how it interacts with `ICQ`.
+
+<!-- TOC 2 -->
+- [Concepts](#concepts)
+  - [Oracle](#oracle)
+  - [Interchain Queries (ICQ)](#interchain-queries-icq)
+    - [Note](#note)
```

Comment:
The TOCs should be auto-generated using the `make update-tocs` command.

That command looks for a `<!-- TOC -->` comment in an `.md` file and generates the table of contents from the section headings. One or two numbers can be provided to control the levels of headers to involve. By default, entries are made for the level 2 and 3 headers. Providing a single number (e.g. `<!-- TOC 2 -->`) changes the maximum level from 3 to whatever was provided. So with `TOC 2` you only get the level 2 headers; with `TOC 4` you'd get all 2, 3, and 4 levels. Providing two numbers (e.g. `<!-- TOC 1 5 -->`) sets both the min and max levels to use. In that example, you'd get entries for all level 1, 2, 3, 4, and 5 headers.

Usually you don't want the level 1 header because there's only one in the file that applies to the whole file and is also above the TOC.

For this one, `TOC 2` is probably what's wanted since we don't really need an entry for that note.

It looks like you've got the `<!-- TOC ... -->` comments in the files just fine, but manually created them. So you'll want to run `make update-tocs` and make sure you've got all the entries you think should be in the TOC for that file.

Reply:
Note that `make update-tocs` will also fix a missing entry in one of the marker spec docs. That's what's nice about having them auto-generated. Go ahead and include that change in this PR.

Reply:
Hm, I ran the update-toc.sh script manually. I'll run it through make from now on.